lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py

```diff
@@ -16,33 +17,76 @@
 HTTPTIMEOUT = (3.05, 1)
 
 
+# Duplicated in delivery_status_processor.
+def validate_twilio_event(event):
+    logger.info("validating twilio vetext forwarder event")
+    try:
+        ssm_client = boto3.client("ssm", "us-gov-west-1")
+        auth_ssm_key = os.getenv("TWILIO_AUTH_TOKEN_SSM_NAME", "")
+        if not auth_ssm_key:
+            logger.error("TWILIO_AUTH_TOKEN_SSM_NAME not set")
+            return False
+
+        response = ssm_client.get_parameter(
+            Name=auth_ssm_key,
+            WithDecryption=True
+        )
+        auth_token = response.get("Parameter").get("Value")
+        signature = event["headers"].get("x-twilio-signature", "")
+    except Exception as e:
+        logger.error("SMS retrival error %" % e)
```

Comment:
should be  `logger.error("SMS retrival error %s" % e)` to clear the CodeQL issue